src/components/Page.jsx

```diff
@@ -35,9 +36,15 @@ const Page = () => {
       }, []);
 
     return (
-        <article>
-            <StepTracker steps={steps} activeStep={activeStep}/>
-            <Card activeStep={activeStep} handleBack={handleBack} handleNext={handleNext}/>
+        <article className='page-container'>
+            <header className='header'>
+
```

Comment:
Minor, but the semantic html could be improved.

<main>
  <header/>
  <section/>
  <footer/>
</main>